mappings/net/minecraft/network/packet/s2c/play/GameStateChangeS2CPacket.mapping

```diff
@@ -1,7 +1,25 @@
 CLASS net/minecraft/class_2668 net/minecraft/network/packet/s2c/play/GameStateChangeS2CPacket
 	FIELD field_12198 value F
 	FIELD field_12199 reason Lnet/minecraft/class_2668$class_5402;
+	FIELD field_25645 NO_RESPAWN_BLOCK Lnet/minecraft/class_2668$class_5402;
+	FIELD field_25646 RAIN_STARTED Lnet/minecraft/class_2668$class_5402;
+	FIELD field_25647 RAIN_STOPPED Lnet/minecraft/class_2668$class_5402;
+	FIELD field_25648 GAME_MODE_CHANGED Lnet/minecraft/class_2668$class_5402;
+	FIELD field_25649 GAME_WON Lnet/minecraft/class_2668$class_5402;
+	FIELD field_25650 DEMO_MESSAGE_SHOWN Lnet/minecraft/class_2668$class_5402;
+	FIELD field_25651 PROJECTILE_HIT_PLAYER Lnet/minecraft/class_2668$class_5402;
```

Comment:
```suggestion
	FIELD field_25651 ARROW_HIT_PLAYER Lnet/minecraft/class_2668$class_5402;
```
[wiki.vg](https://wiki.vg/Protocol#Change_Game_State) describes this as an _arrow_ hitting the player. Is there anywhere else that states that other projectiles cause this packet to be sent?

Reply:
Don't think wiki.vg is that reliable on new contents. For example, you can see that mojang writes registry to a compound tag from code while wiki.vg was initially broken by the dfu codec registry update

Reply:
Fyi wiki.vg's license is cc0 sharealike while yarn is cc0 (no sharealike). As a result, you cannot use wiki.vg's content as your evidence/reason for naming a member.

Reply:
If this is the case, we should amend our `contributing.md` to explicitly prohibit use of wiki.vg for reference